dw-select-dialog.js

```diff
@@ -114,7 +114,7 @@ export class DwSelectDialog extends DwSelectBaseDialog {
         }
 
         :host([single-select]) .main-content {
-          padding-bottom: 12px;
+          padding-bottom: var(--single-select-padding-bottom, 12px);
```

Comment:
This CSS property naming convention seems to have an issue. It doesn't prefix element name